google_cdn-external/variables.tf

```
@@ -115,3 +115,15 @@ variable "quic_override" {
   type        = string
   default     = "DISABLE"
 }
+
+variable "custom_response_headers" {
+  description = "Specifies custom response headers"
+  type        = list(string)
+  default     = null
+}
+
+variable "custom_request_headers" {
+  description = "Specifies custom response headers"
+  type        = list(string)
+  default     = null
```

```suggestion
  default     = []
```
concat doesn't seem to evaluate null properly
```
> concat(["test"], null)

> ╷
│ Error: Invalid function argument
│
│   on <console-input> line 1:
│   (source code not available)
│
│ Invalid value for "seqs" parameter: argument must not be null.
╵

> concat(["test"], [])
[
  "test",
]
```

---

That is a great catch @jasonthomas